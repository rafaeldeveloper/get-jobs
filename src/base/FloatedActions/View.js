import React from 'react'
import { Button, Icon } from 'semantic-ui-react'

export default ({ data, actions }) => (
    <Button.Group>
        {
            actions ? actions.map(action => (
                <Button icon onClick={() => action.execute(data)}>
                    <Icon name={action.icon} />
                </Button>
            ))

            :

            <div>No Actions</div>
        }
    </Button.Group>
)