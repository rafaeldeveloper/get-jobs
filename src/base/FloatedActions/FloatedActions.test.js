import React from 'react';
import FloatedActions from 'base/FloatedActions';
import renderer from 'react-test-renderer';




test('FloatedActions renders correctly', () => {

    const component = renderer.create(
        <FloatedActions/>
    );

    let tree = component.toJSON();
    
    expect(tree).toMatchSnapshot();
''
});

