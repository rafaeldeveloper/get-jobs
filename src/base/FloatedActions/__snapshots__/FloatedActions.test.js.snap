// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FloatedActions renders correctly 1`] = `
<div
  className="ui buttons"
>
  <div>
    No Actions
  </div>
</div>
`;
